plugins/operator-examples/__init__.py

```diff
@@ -815,6 +815,74 @@ def execute(self, ctx):
         ctx.ops.set_view(view=view)
 
 
+class ExampleComplexExecution(foo.Operator):
+    @property
+    def config(self):
+        return foo.OperatorConfig(
+            name="example_complex_execution",
+            label="Example Complex Execution",
+            allow_immediate_execution=True,
+            allow_delegated_execution=True,
+            default_choice_to_delegated=True,
+            resolve_execution_options_on_change=False,
```

Comment:
@ritch repeating my thought from [here](https://github.com/voxel51/fiftyone-plugins/pull/183#issuecomment-2499653722):

It would be ideal if we didn't have to manually specify `resolve_execution_options_on_change=False` here as we've already implicitly communicated this by using `OperatorConfig` rather than implementing `resolve_execution_options()`.